src/runtime/config/configuration-remote.toml.in

```diff
@@ -242,6 +242,16 @@ disable_new_netns = true
 # See: https://pkg.go.dev/github.com/kata-containers/kata-containers/src/runtime/virtcontainers#ContainerType
 sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 
+# If enabled, the runtime will attempt to determine appropriate sandbox size (memory, CPU) before booting the virtual machine. In
+# this case, the runtime will not dynamically update the amount of memory and CPU in the virtual machine. This is generally helpful
+# when a hardware architecture or hypervisor solutions is utilized which does not support CPU and/or memory hotplug.
+# Compatibility for determining appropriate sandbox (VM) size:
+# - When running with pods, sandbox sizing information will only be available if using Kubernetes >= 1.23 and containerd >= 1.6. CRI-O
+#   does not yet support sandbox sizing annotations.
+# - When running single containers using a tool like ctr, container sizing information will be available.
+# Note: the remote hypervisor uses the peer pod config to determine the sandbox size, so requires this to be set to true
```

Comment:
May be when there is an alternate implementation, during that time this can be changed ?
For now we only have peer-pods where dynamic resources handling is not possible. So imho, we should leave it as is and change it when there is a concrete alternative implementation. This is of course my personal view :-)

Reply:
@bpradipt a few minutes after I wrote that comment I changed my mind, and yes,  we better keep `configuration-remote` which defaults to the only implementation available: peer pods. So if in the future we end up with another implementation then we can start differentiating like `configuration-remote-caa`, `configuration-remote-foo`...etc